index.d.ts

```diff
@@ -0,0 +1,25 @@
+export type TAlgorithm = 'HS256' | 'HS384' | 'HS512' | 'RS256';
+
+export interface IOptions {
+  header: any;
+};
+
+/**
+ * Decode jwt
+ * @param token
+ * @param key
+ * @param noVerify
+ * @param algorithm default to HS256
```

Comment:
Why add the jsdoc here since it's already in the code?

Reply:
You are right, it's not necessary :+1:  I will bump the version to 0.5.2 as I would like to make a feat request for the 0.6.0 in a PR to come.